quality/data_availability_dlt.sql

```diff
@@ -107,25 +107,15 @@ CREATE OR REFRESH LIVE TABLE data_availability
         GROUP BY 1
     ),
 
-    energy_consum as (
-        SELECT country_name, 
-            min(year) as renew_energy_consum_earliest_year,
-            max(year) as renew_energy_consum_latest_year
-        FROM indicator.energy_consumption
-        WHERE primary_renewable_consumption_share is not null
-        GROUP BY 1
-    ),
-
-    energy_mix as (
-        SELECT country_name, 
-            min(year) as solar_wind_gen_earliest_year,
-            max(year) as solar_wind_gen_latest_year
+    energy_generation as (
+        select country_name,  
+        min(year) as energy_generation_earliest_year,
+        max(year) as energy_generation_latest_year
```

Comment:
Also add wind_and_solar_gen_earliest_year, wind_and_solar_gen_latest_year as they are of specific interest to the climate team.

Reply:
addressed!